internal/cmd/users.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/users"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func ListServerUsersTable(response core.APIResponse) string {
+	var responseContent users.ListResponse
+	util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
+
+	var serverUsersTable strings.Builder
+	headers := []string{"email", "name", "server-role", "courses"}
+
+	serverUsersTable.WriteString(strings.Join(headers, "\t") + "\n")
+
+	for i, user := range responseContent.Users {
+		coursesJSON := util.MustToJSONIndent(user.Courses)
+
+		serverUsersTable.WriteString(user.Email)
+		serverUsersTable.WriteString("\t")
+		serverUsersTable.WriteString(user.Name)
+		serverUsersTable.WriteString("\t")
+		serverUsersTable.WriteString(user.Role.String())
+		serverUsersTable.WriteString("\t")
+		serverUsersTable.WriteString(coursesJSON)
+
+		// Only add a newline if this is not the last user
+		if i < len(responseContent.Users)-1 {
```

Comment:
Instead of doing there here (and computing a len),
just do it before the row and check for `i > 0`.